README.md

```diff
@@ -1,170 +1,107 @@
+<br>The BigBang of our Android projects.
 
-The easiest way to copy them in the repo folder, is by downloading the GDrive folder and running the
-following:
+Our architecture divides the typical layers of an Android application into simple coherent modules, which are completely independent from one another:
+* UI
+* Services
+* Database
 
-```bash
-./copy-assets.sh path-of-assets-folder
-```
+In order to keep modules independent, we decoupled the behavior required for each layer into a single module, where everything comes together smoothly, which we call `core`. This not only allows you to choose which modules to use, but also gives you independence from any service or database oriented libraries we use.
 
-If you want to do it manually, or understand the process in order to add keys, see the following
-subsections.
+Learn more about the project on the [BigBang Wiki](https://github.com/xmartlabs/bigbang/wiki) or by reading the Javadocs.
 
-### Key stores
+## Getting started
 
-`debug.keystore` and `release.keystore` files need to be present in the `app` folder. Also the file
-`app/keystore.properties` is needed with the following inside (replace where needed):
+The first step is to include the Core library into your project as a Gradle compile dependency:
 
-```data
-store_password=STORE_PASSWORD
-key_alias=KEY_ALIAS
-key_password=KEY_PASSWORD
+```groovy
+compile 'com.xmartlabs.bigbang:core:0.1.10'
 ```
 
-If you need to create them, run the following in the `app` folder:
+Then, add jCenter to the buildscript repositories:
 
-```bash
-keytool -genkey -v -keystore debug.keystore -alias KEY_ALIAS -keyalg RSA -keysize 2048 -validity 10000
-keytool -genkey -v -keystore release.keystore -alias KEY_ALIAS -keyalg RSA -keysize 2048 -validity 10000
+```groovy
+buildscript {
+    repositories {
+        jcenter()
+    }
+}
 ```
 
-### Generic keys file
-
-For the following services, add the files `app/src/production/res/values/keys.xml` and
-`app/src/staging/res/values/keys.xml` (for each environment) with this empty content:
-
-```xml
-<?xml version="1.0" encoding="utf-8" ?>
-<resources>
-    <string name="SOME_SERVICE" translatable="false">SOME_SERVICE_TOKEN</string>
-</resources>
+Also, you'll need to add the following repositories:
+
+```groovy
+repositories {
+    flatDir {
+        dirs 'libs'
+    }
+    maven { url "https://jitpack.io" }
+    maven { url 'https://maven.fabric.io/public' }
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+}
 ```
 
-### Fabric
-
-In order to use Fabric, create a file `app/fabric.properties` with the content (replace with the
-right values):
-
-```data
-apiSecret=YOUR_BUILD_SECRET
-apiKey=YOUR_API_KEY
+Then, include any of the following compile dependencies to add an specific module to your app:
+```groovy
+compile 'com.xmartlabs.bigbang:dbflow:0.1.4'
```

Comment:
But aren't them so tightly coupled that you need to know what versions are compatible with what versions? Just like rxandroid and rxjava? And, in consequence, e.g., certain versions of dbflow won't be compatible with certain versions of core in the future?